.github/workflows/build.yml

```
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 레포지토리 체크아웃 중..
+        uses: actions/checkout@v2
+
+      - name: node 설치중..
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18.18.2'
```

각 액션들 버전을 올리면 좋을 거 같아요 ~ 
```suggestion
    steps:
      - name: 레포지토리 체크아웃 중..
        uses: actions/checkout@v4

      - name: node 설치중..
        uses: actions/setup-node@v4
        with:
          node-version: '18.18.2'
```